### PR TITLE
[WIP]RHDEVDOCS-3856 Establish Jenkins category under the CI/CD

### PR DESCRIPTION
**Pull request by rolfedh**
Purposes:
- In the left nav, under "CI/CD", create a "Jenkins" node.
- Where appropriate, use the "OpenShift Pipelines" product name instead of "Tekton".

 Aligned team: Dev Tools
- For branches: 4.11+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3856
- Direct link to doc preview: https://deploy-preview-43039--osdocs.netlify.app/openshift-enterprise/latest/cicd/jenkins/migrating-from-jenkins-to-openshift-pipelines.html
- SME review: @akram @rupalibehera  (both approved)
- QE review: Not needed - no changes to the technical content.
- Peer review: @sounix000 and @bburt-rh (both approved)
- Reviews complete. Please merge now.

### CODE REVIEW

File: modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc [hunk 1]
```diff
@@ -34,8 +34,8 @@ pipeline {
 }
 ----
 
-== Tekton pipeline
-In Tekton, the equivalent example of the Jenkins pipeline comprises of three tasks, each of which can be written declaratively using the YAML syntax:
+== OpenShift Pipelines pipeline
+In OpenShift Pipelines, the equivalent example of the Jenkins pipeline comprises of three tasks, each of which can be written declaratively using the YAML syntax:
```
Comment (rolfedh) [inline modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:38]:
```suggestion
In OpenShift Pipelines, the equivalent example of the Jenkins pipeline is composed of three tasks, each of which can be written declaratively using the YAML syntax:
```

File: modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc
+
+[id="jt-comparison-of-jenkins-openshift-pipelines-execution-models_{context}"]
+= Comparison of Jenkins and OpenShift Pipelines execution models
+
+Jenkins and OpenShift Pipelines offer similar functions but are different in architecture and execution. This section outlines a brief comparison of the two execution models.
```
Comment (bburt-rh) [inline modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc:8]:
```suggestion
Jenkins and OpenShift Pipelines offer similar functions but are different in architecture and execution. 
```
Consider omitting the meta, pointing language that already stated in the heading that follows. I generally find that  phrasing like "in this section" and often the content that follows can nearly always be cut.

File: modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+.Comparison of execution models in Jenkins and OpenShift Pipelines
+[cols="1,1",options="header"]
+|===
+|Jenkins|OpenShift Pipelines
+|Jenkins has a control node. Jenkins executes pipelines and steps centrally, or orchestrates jobs running in other nodes.|OpenShift Pipelines is serverless and distributed, and there is no central dependency for execution.
+|The containers are launched by the control node through the pipeline.|OpenShift Pipelines adopts a 'container-first' approach, where every step is executed as a container running in a pod (equivalent to nodes in Jenkins).
```
Comment (bburt-rh) [inline modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc:15]:
```suggestion
|Containers are launched by the control node through the pipeline.|OpenShift Pipelines adopts a 'container-first' approach, where every step is executed as a container running in a pod (equivalent to nodes in Jenkins).
```

File: modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+|Extensibility is achieved using plugins.|Extensibility is achieved using tasks in Tekton Hub, or by creating custom tasks and scripts.
```
Comment (bburt-rh) [inline modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc:16]:
```suggestion
|Extensibility is achieved using plugins.|Extensibility is achieved by using tasks in Tekton Hub or by creating custom tasks and scripts.
```
- Should `Tekton`  here be changed to `OpenShift Pipelines`?

File: modules/jt-examples-of-common-use-cases.adoc [hunk 5]
```diff
@@ -50,7 +50,7 @@ node('maven') {
 
 ----
 
-.Example: Compile and build an image and deploy it to OpenShift using maven in Tekton.
+.Example: Compile and build an image and deploy it to OpenShift using maven in OpenShift Pipelines.
```
Comment (bburt-rh) [inline modules/jt-examples-of-common-use-cases.adoc:53]: If this is Apache Maven, then all instances of `maven` should be capitalized https://maven.apache.org/. Also, on first use it should use the whole product name (`Apache Maven`).

File: modules/jt-examples-of-common-use-cases.adoc [hunk 6]
```diff
@@ -151,14 +151,14 @@ spec:
...
 
-== Sharing reusable code in Jenkins and Tekton
+== Sharing reusable code in Jenkins and OpenShift Pipelines
 Jenkins link:https://www.jenkins.io/doc/book/pipeline/shared-libraries/[shared libraries] provide reusable code for parts of Jenkins pipelines. The libraries are shared between link:https://www.jenkins.io/doc/book/pipeline/jenkinsfile/[Jenkinsfiles] to create highly modular pipelines without code repetition.
 
-Although there is no direct equivalent of Jenkins shared libraries in Tekton, you can achieve similar workflows by using tasks from the link:https://hub.tekton.dev/[Tekton Hub], in combination with custom tasks and scripts.
\ No newline at end of file
+Although there is no direct equivalent of Jenkins shared libraries in OpenShift Pipelines, you can achieve similar workflows by using tasks from the link:https://hub.tekton.dev/[Tekton Hub], in combination with custom tasks and scripts.
```
Comment (bburt-rh) [inline modules/jt-examples-of-common-use-cases.adoc:164]:
```suggestion
Although there is no direct equivalent of Jenkins shared libraries in OpenShift Pipelines, you can achieve similar workflows by using tasks from the link:https://hub.tekton.dev/[Tekton Hub] in combination with custom tasks and scripts.
```
- no comma necessary here, I think

File: modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc [hunk 7]
```diff
@@ -38,16 +38,16 @@ You can initiate a PipelineRun or a TaskRun with a set of inputs such as paramet
...
+In Jenkins, there is no direct equivalent of OpenShift Pipelines workspaces. You can think of the control node as a workspace, as it stores the cloned code repository, build history, and artifacts. In situations where a job is assigned to a different node, the cloned code and the generated artifacts are stored in that node, but the build history is maintained by the control node.
 ====
 
 == Mapping of concepts
-The building blocks of Jenkins and Tekton are not equivalent, and a comparison does not provide a technically accurate mapping. The following terms and concepts in Jenkins and Tekton correlate in general:
+The building blocks of Jenkins and OpenShift Pipelines are not equivalent, and a comparison does not provide a technically accurate mapping. The following terms and concepts in Jenkins and OpenShift Pipelines correlate in general:
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc:45]: Updated. Thanks.

File: modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc:8]: Updated. Thanks.

File: modules/jt-extending-openshift-pipelines-capabilities-using-custom-tasks-and-scripts.adoc [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
-// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+// jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc
 
-[id="jt-extending-tekton-capabilities-using-custom-tasks-and-scripts_{context}"]
-= Extending Tekton capabilities using custom tasks and scripts
+[id="jt-extending-openshift-pipelines-capabilities-using-custom-tasks-and-scripts_{context}"]
+= Extending OpenShift Pipelines capabilities using custom tasks and scripts
 
-In Tekton, if you do not find the right task in Tekton Hub, or need greater control over tasks, you can create custom tasks and scripts to extend Tekton's capabilities.
+In OpenShift Pipelines, if you do not find the right task in Tekton Hub, or need greater control over tasks, you can create custom tasks and scripts to extend OpenShift Pipelines's capabilities.
```
Comment (bburt-rh) [inline modules/jt-extending-openshift-pipelines-capabilities-using-custom-tasks-and-scripts.adoc:8]:
```suggestion
In OpenShift Pipelines, if you do not find the right task in Tekton Hub, or need greater control over tasks, you can create custom tasks and scripts to extend the capabilities of OpenShift Pipelines.
```
Per IBM SG: `Do not use 's to show the possessive form of abbreviations or product names.`

File: modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc (hunk 1 above)
Comment (bburt-rh) [inline modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:38]:
```suggestion
In OpenShift Pipelines, the equivalent example of the Jenkins pipeline is comprised of three tasks, each of which can be written declaratively using the YAML syntax:
```
`comprises of` is incorrect usage

File: modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc [hunk 9]
```diff
@@ -90,9 +90,9 @@ spec:
     workingDir: $(workspaces.source.path)
 ----
 
-You can combine the three tasks sequentially to form a Tekton pipeline:
+You can combine the three tasks sequentially to form a OpenShift Pipelines pipeline:
```
Comment (bburt-rh) [inline modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:93]:
```suggestion
You can combine the three tasks sequentially to form a pipeline in OpenShift Pipelines:
```

Comment (rolfedh) [reply modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:93]: Updated. Thanks.

File: modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:38]: Updated. Thanks.

Comment (rolfedh) [reply modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:38]: Updated. Thanks.

File: modules/jt-extending-openshift-pipelines-capabilities-using-custom-tasks-and-scripts.adoc (hunk 8 above)
Comment (rolfedh) [reply modules/jt-extending-openshift-pipelines-capabilities-using-custom-tasks-and-scripts.adoc:8]: Updated. Thanks.

File: modules/jt-examples-of-common-use-cases.adoc (hunk 6 above)
Comment (rolfedh) [reply modules/jt-examples-of-common-use-cases.adoc:164]: Updated. Thanks.

File: modules/jt-examples-of-common-use-cases.adoc (hunk 5 above)
Comment (rolfedh) [reply modules/jt-examples-of-common-use-cases.adoc:53]: Updated. Thanks.

File: modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc (hunk 4 above)
Comment (rolfedh) [reply modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc:16]: Thanks. Updated. In this case, Tekton Hub is a site name that does not need to be changed. There is no equivalent hub for OpenShift Pipelines.

File: modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc [hunk 10]
```diff
@@ -27,7 +28,7 @@ Tekton uses the link:https://yaml.org/[YAML] syntax for declarative pipelines an
 ====
 You can initiate a PipelineRun or a TaskRun with a set of inputs such as parameters and workspaces, and the execution results in a set of outputs and artifacts.
```
Comment (akram) [inline modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc:29]: impersonate

File: modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc [hunk 11]
```diff
@@ -38,16 +39,16 @@ You can initiate a PipelineRun or a TaskRun with a set of inputs such as paramet
 +
 [NOTE]
 ====
-In Jenkins, there is no direct equivalent of Tekton workspaces. You can think of the control node as a workspace, as it stores the cloned code repository, build history, and artifacts. In situations where a job is assigned to a different node, the cloned code and the generated artifacts are stored in that node, but the build history is maintained by the control node.
+In Jenkins, there is no direct equivalent of OpenShift Pipelines workspaces. You can think of the control node as a workspace, as it stores the cloned code repository, build history, and artifacts. In situations where a job is assigned to a different node, the cloned code and the generated artifacts are stored in that node, but the build history is maintained by the control node.
```
Comment (akram) [inline modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc:42]: Replace "control node" by "controller node" which is the more appropriate terminology.

File: modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc [hunk 12]
```diff
@@ -34,10 +35,11 @@ pipeline {
 }
 ----
 
-== Tekton pipeline
-In Tekton, the equivalent example of the Jenkins pipeline comprises of three tasks, each of which can be written declaratively using the YAML syntax:
+== OpenShift Pipelines pipeline
 
-.Example `build` task
+To create a pipeline in OpenShift Pipelines that is equivalent to the preceding Jenkins pipeline, you create the following three tasks:
```
Comment (akram) [inline modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:40]: `you create the following three tasks ` replace by `the following three tasks can be created`

File: modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+[id="jt-comparison-of-jenkins-openshift-pipelines-execution-models_{context}"]
+= Comparison of Jenkins and OpenShift Pipelines execution models
+
+Jenkins and OpenShift Pipelines offer similar functions but are different in architecture and execution.
+
+.Comparison of execution models in Jenkins and OpenShift Pipelines
+[cols="1,1",options="header"]
+|===
+|Jenkins|OpenShift Pipelines
+|Jenkins has a control node. Jenkins executes pipelines and steps centrally, or orchestrates jobs running in other nodes.|OpenShift Pipelines is serverless and distributed, and there is no central dependency for execution.
```
Comment (akram) [inline modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc:15]: use `controller node` instead

File: modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc [hunk 14]
```diff
@@ -1,11 +1,12 @@
...
+// jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc
 
-[id="jt-migrating-a-sample-pipeline-from-jenkins-to-tekton_{context}"]
-= Migrating a sample pipeline from Jenkins to Tekton
+:_content-type: PROCEDURE
+[id="jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines_{context}"]
+= Migrating a sample pipeline from Jenkins to OpenShift Pipelines
 
-This section provides equivalent examples of pipelines in Jenkins and Tekton and helps you to migrate your build, test, and deploy pipelines from Jenkins to Tekton.
+You can use the following equivalent examples to help migrate your build, test, and deploy pipelines from Jenkins to OpenShift Pipelines.
```
Comment (akram) [inline modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:9]: is it relevant to use a `placeholder` for `OpenShift Pipelines` ? This way we can have a different documentation between enterprise and community.

**Review by akram — CHANGES_REQUESTED** [review]

File: cicd/jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc [hunk 15]
```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+//Jenkins-Tekton-Migration
+[id="migrating-from-jenkins-to-openshift-pipelines_{context}"]
+= Migrating from Jenkins to OpenShift Pipelines or Tekton
+include::_attributes/common-attributes.adoc[]
+:context: migrating-from-jenkins-to-openshift-pipelines
+
+toc::[]
+
+You can migrate your CI/CD workflows from Jenkins to xref:../../cicd/pipelines/understanding-openshift-pipelines.adoc#understanding-openshift-pipelines[Red Hat OpenShift Pipelines], a cloud-native CI/CD experience based on the Tekton project.
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply cicd/jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc:10]: Thanks, @akram. In the past, we would have normally used impersonal wording. However, now, we have newer style guidance to refer to the reader directly as "you."

File: modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc [hunk 16]
```diff
@@ -1,11 +1,12 @@
...
+// jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc
 
-[id="jt-comparison-of-jenkins-and-tekton-concepts_{context}"]
-= Comparison of Jenkins and Tekton concepts
+:_content-type: CONCEPT
+[id="jt-comparison-of-jenkins-and-openshift-pipelines-concepts_{context}"]
+= Comparison of Jenkins and OpenShift Pipelines concepts
 
-This section summarizes the basic terms used in Jenkins and Tekton, and compares the equivalent terms.
+You can review and compare the following equivalent terms used in Jenkins and OpenShift Pipelines.
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc:9]: IBM Style Guide, which we follow: "Write clearly and directly. Wherever possible, use the second person ('you')."

File: modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc (hunk 11 above)
Comment (rolfedh) [reply modules/jt-comparison-of-jenkins-and-openshift-pipelines-concepts.adoc:42]: Thanks. Updated.

File: modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc (hunk 12 above)
Comment (rolfedh) [reply modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:40]: Red Hat documentation prefers using the second-person pronoun, "you."

File: modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc (hunk 14 above)
Comment (rolfedh) [reply modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc:9]: That's an interesting suggestion. I'll leave it to Souvik Sarkar (@sounix000), the primary writer for OpenShift Pipelines, whether to implement that change. If so, he'll do it in a separate doc task.

**Review by rupalibehera — APPROVED** [review]

**Comment by gabemontero** [comment]
Per our recent conversation the general direction here looks good @rolfedh 

So yeah, as the next step (separate PR per our discussion) we would like to see "Configure Jenkins images" from https://deploy-preview-43039--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using_images/images-other-jenkins.html move under CI/CD -> Jenkins, *BEFORE* the migrating article.

And similarly "Jenkins agent" https://deploy-preview-43039--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/using_images/images-other-jenkins-agent.html would slot in after "Configuring.." but before "Migrating..."

Then "Images -> Using Images" would become "Images -> Using Source-to-image" with the remaining 3 articles that are source to image centric (their titles might need some tweaking").

**Pull request merged**